.pipelines/templates/release.yml

```diff
@@ -35,12 +35,17 @@ jobs:
   variables:
     version: $[ dependencies.Version.outputs['setVer.version'] ]
     REGISTRY_NAME: '$(ACR_REGISTRY_NAME).azurecr.io/'
-    IMAGE_PREFIX: 'public/'
-    TAG: ':$(version)'
+    IMAGE_PREFIX: "public/"
   steps:
-    - template: ./build-containers.yml
+    - template: ./build-and-push-containers.yml
```

Comment:
maybe just add a comment why we're doing this twice. either here or at the top of the yaml as in the CI/PR pipelines.

Reply:
sure!